### ISSUE TITLE
Disable/Override Input Validation

### ISSUE THREAD

**Issue by j-ki** [soflyy/oxygen-bugs-and-features#1698]
**Describe the feature you'd like to see included in Oxygen.**
*A clear, detailed description of the feature you're requesting.*

I would like a way to either disable or override input validation (e.g. "Value may NOT contain quotes and '<', '>' characters. These were auto converted to corresponding HTML enteties (sic).").

This could either be a override for a single case or a site-wide disable option.

**What are the use cases for this feature?**
*e.g. Accessibility, e-commerce store that sells bicycles, etc.*

For links, I'd like to create a 'onclick' attribute that fires JavaScript. For example, FaceBook's AddToCart: fbq('track', 'AddToCart');.

**Examples of this feature or functionality.**
*If another product has done it well, provide examples here.*

**Comment by KittenCodes** [comment]
Are you able to use Oxygen's Attributes for this?

**Comment by j-ki** [comment]
No, I cannot put in JavaScript in Oxygen's attributes. That's where I want to put in Facebook functions. But without the ability to enter special characters like quotes, I cannot.

**Comment by YammYcoding** [comment]
+1

**Comment by KittenCodes** [comment]
> No, I cannot put in JavaScript in Oxygen's attributes. That's where I want to put in Facebook functions. But without the ability to enter special characters like quotes, I cannot.

Understood, thank you.

**Comment by mallettejp** [comment]
+1

I'd love to be able to use Alpine without having to put in code blocks
